frontend/package.json

```diff
@@ -43,8 +43,10 @@
 		"@mui/material": "^5.14.0",
 		"@mui/styles": "^5.3.0",
 		"@mui/x-date-pickers": "^5.0.8",
-		"@rjsf/core": "^3.2.1",
-		"@rjsf/material-ui": "^3.2.1",
+		"@rjsf/core": "^5.18.4",
+		"@rjsf/utils": "^5.18.4",
```

Comment:
is @rjsf/utils used?

Reply:
Yes, it's needed. It's a dependency for rjsf/mui to work.

Reply:
Refer to Prerequisites in https://www.npmjs.com/package/@rjsf/mui